src/UI/SDK/package.json

```diff
@@ -40,8 +40,13 @@
     "typescript": "~3.7.5"
   },
   "dependencies": {
+    "@angular/common": "^12.1.1",
+    "@angular/core": "^12.1.1",
+    "@ordercloud/angular-sdk": "^1.1.0",
     "lodash": "^4.17.21",
-    "ordercloud-javascript-sdk": "4.5.4"
+    "ngx-cookie": "^4.1.2",
```

Comment:
Getting the following errors:
Error: The target entry-point "@ordercloud/angular-sdk" has missing dependencies:
 - @nguniversal/express-engine/tokens

I think it might be better to define these as peer dependencies since its assumed those dependencies will be part of the project consuming the headstart sdk package

Reply:
I was able resolve the issue by adding a few more dependencies, however I was having some trouble in using the peer dependencies, but we can revisit this again at a later date.